src/forms/redux-form/CalendarComponent.jsx

```diff
@@ -8,8 +8,10 @@ import CalendarComponent from '../CalendarComponent';
  * @return {Component} TextInput
  */
 const ReduxFormCalendarComponent = props => {
-	const { meta, input, ...other } = props;
-	return <CalendarComponent error={meta.error} {...input} {...other} />;
+	const { meta, input, validateBeforeTouched, ...other } = props;
+	const error = (validateBeforeTouched || meta.touched) ? meta.error : null;
```

Comment:
looks like `meta` is now a required prop? should get a proptype if so, and I suspect it will break some existing implementations?